tests/core/test_kubeutil.py

```diff
@@ -68,6 +68,7 @@ def test_deployment_name_k8s_1_8(self):
         self.assertEqual('front-end', self.kube.get_deployment_for_replicaset('front-end-768dd754b7'))
 
 
+# Creator Tags tests for k8 version < 1.9: getting them from the annotation
```

Comment:
The documentation should be conform to the [PEP257](https://www.python.org/dev/peps/pep-0257/)
Like:
```python
class MyClass(object):
    """
    My cool comment
    """
```

Then, I could use some tools to directly build the documentation, they'll call `MyClass.__doc__` to get `My cool comment`